skbio/io/format/tests/test_embed.py

```diff
@@ -25,6 +29,28 @@
     _protein_to_vector, _vector_to_protein
 )
 
+class TestWriteError(TestCase):
+    def test_write_function(self):
+        embed_file = os.path.join('data', 'tiny_embedding_file.npz')
+        embeddings = np.load(embed_file, allow_pickle=False)
+        emb_list = embeddings.values() 
+        sequences = np.loadtxt('data/pdb_hits.txt', dtype=str)
```

Comment:
TODO : add comments here describing the pdb_hits file.

Also, try using `skbio.util.get_data_path` here. It is more robust to deployment in environment.

```suggestion
        sequences = get_data_path('pdb_hits.txt')  # TODO : make sure that get_data_path is correct
```